web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.ts

```diff
@@ -102,8 +102,8 @@ export class IngestUiComponent implements OnInit {
 
   getStep(flow): Step {
     this.step.options.inputFilePath = this.folder;
-    this.step.options.transformParams = `entity-name=${this.step.targetEntity},flow-name=${flow.name}`;
-    this.step.options.outputCollections = `${this.step.targetEntity}`;
+    this.step.options.transformParams = `entity-name=${this.step.options.targetEntity},flow-name=${flow.name}`;
+    this.step.options.outputCollections = `${this.step.options.targetEntity}`;
```

Comment:
Did targetEntity move under the options?

Reply:
Yes